live-examples/html-examples/embedded-content/css/iframe.css

```diff
@@ -0,0 +1,15 @@
+footer {
+    font-size: .8rem;
+    position: absolute;
+    bottom: 10px;
+    right: 10px;
+}
+
+iframe {
+    border: 1px solid black;
```

Comment:
Suggestions: I would set the width of the `iframe` here to 100%

Reply:
Added `width: 100%` to `iframe`.